README.md

```diff
@@ -24,7 +24,7 @@ SUNRISE comes with some read-only data set by default that you can use. But if y
 Once you have created your commercetools project and populated it with data, follow the next steps to connect to it.
 
 ### 1. Create an API client for a SPA
-In the [Admin Center](https://admin.commercetools.com/), select your project and go to the `Developers` section and then the `API Clients` tab. Now click on the `Add API client` button to display a form. There, enter a descriptive name for your new API client and click on `Select permissions for an API client suited for a mobile or a single page application`, found below the `Permissions` checkboxes.
+In the [Merchant Center](https://mc.commercetools.com/), select your project and go to the `Developer Settings` section and then the `API Clients` tab. Now click on the `Create New API client` button to display a form. There, enter a descriptive name for your new API client and click on `Select Template for an API client suited for a mobile or a single page application`.
```

Comment:
Before 'Developer Settings' there is another menu. People won't find 'Developer Settings' directly by looking at the screen.

Reply:
Ah a dilema. Now you appear directly in the API Clients tab, and we could take advantage of that and skip that part, or simply say "you should find yourself in the API Clients tab". But what if they decide to add more tabs before this one? Unlikely though, as in any case this is the most important tab there will be. What do you think?

Reply:
"Select Template for an API client suited for a mobile or a single page application" <- I cannot find this line in the UI. Are you sure this is what appears in the MC?

Reply:
Let's say I select the correct permissions. What else, then? I know the previous documentation finished there, but let's make it better this time. Let's describe how to submit the form and where to find the necessary information a bit.